mne/decoding/transformer.py

```diff
@@ -715,3 +717,164 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class Filterer(TransformerMixin):
```

Comment:
why are there no red lines? You should use a private function which is called by both the classes -- the one to be deprecated and the new one